map/src/drawer/util/TrackUtils.js

```diff
@@ -0,0 +1,53 @@
+import {toHHMMSS} from "../../context/AppContext";
+
+function getFileName(file) {
+    if (file.name.includes('/')) {
+        return file.name.split('/')[1]
+    } else if (file.local && file.name.includes(':')) {
+        return file.name.split(':')[1]
+    } else {
+        return file.name;
+    }
+}
+
+function updateTextInfo(gpxFiles, ctx) {
```

Comment:
If it's a separate component or context menu?

Reply:
`+`